packer/ansible/agent.yml

```diff
@@ -92,6 +98,12 @@
           get_url:
             url: https://github.com/digitalocean/doctl/releases/download/v1.73.0/doctl-1.73.0-linux-arm64.tar.gz
             dest: /tmp/doctl.tar.gz
+
+        - name: Download and unarchive AWS installer
+          unarchive:
+            src: https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip
+            dest: /tmp
+            remote_src: yes
```

Comment:
This must be the double of lines 71-76 )

Reply:
It has a different arch: aarch64 here and x86_64 in 71-76.

I wanted to use a variable for this but...Archs don't have a common name...
x86_64 == amd64
amd64 == aarch64